doc/readthedocs/sphinx/index.rst

```diff
@@ -16,6 +16,10 @@ Over time, that scope may grow much further.
    api
    schema
    tips_and_tricks/tips_and_tricks
+   acknowledgments/acknowledgments
+   auxiliary-programs/auxiliary-programs
+   essentials/essentials
+   ems-application-guide/ems-application-guide
```

Comment:
Add these four to the main sphinx rst index file.  It could definitely use some organization, but meh for now.